src/containers/AdminCampaignEdit/components/StartCampaignButton.tsx

```diff
@@ -23,8 +25,15 @@ export const StartCampaignButton: React.FC<StartCampaignButtonProps> = (
   const { data, loading } = useGetCampaignStatusQuery({
     variables: { campaignId }
   });
+  const { data: campaignStepsData } = useGetCampaignInteractionStepsQuery({
+    variables: { campaignId }
+  });
   const [startCampaign] = useStartCampaignMutation();
 
+  // manage warning confirmation and confirmation dialog state
+  const [warningIsConfirmed, setWarningConfirmed] = useState<boolean>(false);
```

Comment:
Nit: We should create real conventions around this but I think this is more in line with what we usually do
```suggestion
  const [warningConfirmed, setWarningConfirmed] = useState<boolean>(false);
```